sygnal/apnspushkin.py

```diff
@@ -51,6 +53,12 @@
     labelnames=["pushkin", "code"],
 )
 
+CERTIFICATE_EXPIRATION_GAUGE = Gauge(
+    "sygnal_cert_expiry",
```

Comment:
This name could probably use bikeshedding, let me know if there's a particular pattern we use.

Reply:
be good to identify it's a client cert rather than the server cert. maybe `sygnal_client_cert_expiry`.